server/utils/s3.js

```diff
@@ -112,3 +115,49 @@ export const uploadToS3FromUrl = async (url: string, key: string) => {
     }
   }
 };
+
+export const setS3ObjectAcl = () =>
+  ENABLE_PRIVATE_CONTENT ? 'private' : 'public-read';
+
+export const getUrlForImageProxy = (key: string) =>
+  `/api/images.info?key=${key}`;
+
+export const getSignedImageUrl = async (key: string) => {
+  const s3 = new AWS.S3({
+    accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+    secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
+  });
+  invariant(AWS_S3_UPLOAD_BUCKET_NAME, 'AWS_S3_UPLOAD_BUCKET_NAME not set');
+
+  const params = {
+    Bucket: process.env.AWS_S3_UPLOAD_BUCKET_NAME,
+    Key: key,
+    Expires: 900,
```

Comment:
Is this 900 seconds? 15 minutes…?

Reply:
yes the units are seconds. 900 is the default so I could either omit it entirely, or expose it as a config variable? `AWS_S3_VIEW_EXPIRY`..something like that?

Reply:
Guidance from AWS is to use as short a time as possible. So I think either I leave it as the default (therefore omit the parameter), or shorten it even further. There is no need for this to be long because once the image is loaded, it's in the page. And if the page is reloaded, then a new url is made anyway.

Reply:
Agreed, let's shorten it to 5 minutes.